pagopa/io-app
# chore: [IOBP-525] Fix payment authorization webview on Android devices

Depends on https://github.com/pagopa/io-react-native-login-utils/pull/11

## Short description
This PR updates the library [io-react-native-login-utils](https://github.com/pagopa/io-react-native-login-utils). This new version of the library allows Android users to navigate to other apps while authorizing payment, avoiding closing the webview as soon as the user brings the IO app to the background.

## List of changes proposed in this pull request
- Update `io-react-native-login-utils` depencency

## How to test
Within **Profile > Playgrounds > New Wallet > Payment**, start a new payment flow. During the payment authorization, which is handler by a web view, try to navigate to other apps. Check that you can still finalize the payment.

## Preview

https://github.com/pagopa/io-app/assets/6160324/2b0b9882-5f81-40f9-8159-0e257c7240c2
